reg_tests/CTestList.cmake

```diff
@@ -66,7 +66,7 @@ endfunction(add_test_r_cat)
 # Regression tests
 #=============================================================================
 
-add_test_r_cat(ablForcingEdge 4 11)
+add_test_r_cat(ablForcingEdge 8 11)
```

Comment:
@mchurchf I feel that it would be better to rename this test to `ablNeutralEdge` since the scope of this has expanded beyond just an example of the ABL forcing algorithm. It would be more intuitive for new users and a natural point to start from, especially if they are SOWFA users moving over to Nalu codebase.